docker/README.md

```diff
@@ -65,6 +76,78 @@ $ docker-compose pull
 $ docker-compose up
 ```
 
+## Use of .env file
+The docker-compose file has variables for the image tags. This allows the use of a ``.env`` file.
+By default, ``branch-main`` will be used. The following is an example of a file overriding some services :
+```shell
+FRONTEND_TAG=branch-epic_HOCS-COMP
+CASEWORK_TAG=branch-epic_HOCS-COMP
+WORKFLOW_TAG=branch-epic_HOCS-COMP
+CASE_CREATOR_TAG=branch-epic_HOCS-COMP
+INFO_TAG=branch-epic_HOCS-COMP
+HOCS_DATA_TAG=branch-feature_HOCS-2907-compose-up-all
+HOCS_DATA_REPO=hocs-data
+HOCS_DATA_ELASTIC_REPO=hocs-data-elastic
+```
+The `HOCS_DATA_REPO` can be one of :
```

Comment:
Colons don't generally have leading spaces. Don't make me dust off my Chicago Manual of Style.
```suggestion
The `HOCS_DATA_REPO` can be one of:
```

Reply:
done